develop/TOOLS/DIS/ToolWindows/DisValidatorToolWindow.md

```diff
@@ -6,14 +6,60 @@ uid: DisValidatorToolWindow
 
 If you click *Tool Windows > DIS Validator*, the *DIS Validator* window will appear.
 
-This window lists the results of the latest protocol validation in a tree structure, grouped by severity and category.
+This window lists the results of the latest protocol validation in a treegrid structure, grouped by severity and category.
 
-At the top of the tool window, you can find two buttons:
+At the top of the tool window, you can find the following buttons:
 
 - Click *Validate* to launch a new protocol validation.
-
 - Click *Export* to export the results of the latest validation to a CSV file.
+- Click the ![Show/hide suppressed results](~/develop/images/DisValidatorToolWindowToggleResultsIcon.png) icon to toggle visualization of the results to only show the Active items or all items (i.e., Active, Suppressed and Postponed items).
+
+> [!NOTE]
+> As of DIS v2.41, when you open a main DVE protocol and click *Validate*, DIS will not only validate the main protocol but also all its exported child DVE protocols.
+
+The validator results are listed in a treegrid which groups the validation results by their severity. The treegrid has the following columns:
 
-As of DIS v2.41, when you open a main DVE protocol and click *Validate*, DIS will not only validate the main protocol but also all its exported child DVE protocols.
+- Description: Description: Description of the issue
+- Information icon: This icon is shown in case more information is available about this validator check. Clicking the information icon will open a pop-up window which provided more details about the validator check.
+- Autofix icon: If this icon is displayed, an auto-fix is available. To execute the fix, from the context menu, select either *Fix* > *This error* or *Fix* > *All errors of this type*
+- State: Shows the state of this validation result (Active, Postponed or Suppressed). 
+- Certainty: Specifies whether the validator is certain or uncertain about the validity of the detected issue.
+- Fix Impact: Indicates whether fixing the issues results in a breaking change.
+- Category: Displays the category of the check (e.g. Param or Protocol).
+- Code: Displays the unique code of the validator check.
+- Line: Specifies the line number where the detected issue.
+- Column: Specifies the column number where the issue was detected.
+- DVE: Displays the name of the DVE protocol in case this validation result belongs to a DVE protocol.
 
 ![DIS Validator tool window](~/develop/images/DisValidatorToolWindow.png)
+
+## Suppressing or postponing a validation result
+
+In case a validation result is not valid, it can be suppressed. To suppress a validation result, select the item from the list, right click to open the context menu and select *Suppress...* from the context menu. In the *Suppress Validator Result* window that appears, provide a reason why the result has been suppressed.
```

Comment:
It would be great if we could here explain that we allow suppression on any result for flexibility but that, in theory, only a validation result defined as "Uncertain" should be suppressed. Indeed such "Uncertain" checks require a human to double check them as the automated validation can't be sure. If one would need to suppress a "Certain" result, it probalby means the check is buggy and either should be improved, either should be defined as "Uncertain" and in that case, the proper way to go is to create a validator issue task and postpone it.
The only other (valid? debatable?) use-case where you would suppress a valid and "Certain" remarks is if you consider it's not important enough.

Reply:
Can external users also create a validator issue task, or should they use the "Send feedback" feature instead?

Reply:
DIS now has a "Report Issue..." that brings end-users towards the create UOSA ticket web page.
Personally, I hate it because on there, you then have to select which project, etc which is not very user-friendly to me. But yeah, I gave that feedback to Platform and they told me no improvements were going to be made there and that was still the official way to go :)

Reply:
Right now, we only mention the Send Feedback option in the documentation, so I'll have to add this as well. How could we distinguish between the two options? Perhaps we can say that if you don't know which project to select, you can use Send Feedback instead?

Reply:
Maybe a bit vauge but I would say use "Report Issue..." for reporting issues and use "Send Feedback..." for anything else :) @PedroDebevere, what do you think?

Reply:
Note that the Report issue menu item is currently only available for SLC users only. For other uses only the send feedback menu item is shown. We can add this as an additional note that the report issue is currently not available for other users and that they should use the Send Feedback functionality.